File: contractstudy/src/test/java/contractstudy/collectContracts/common/StaticImportCollector/java/testData/StaticImportClass.java
package contractstudy.collectContracts.common.StaticImportCollector.java.testData;

import com.google.common.base.Preconditions;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public class StaticImportClass {

  static public boolean test(boolean flag) {
    return !flag;
  }

  public int sum(int num1, int num2) {
    return num1 + num2;
  }

  public int compute(int area1, int area2) {
    return sum(area1, area2);
  }

  @NotNull
  public void print(List<String> list) {
    Preconditions.checkArgument(true, "This is an error");
    for (String item : list) {
      System.out.println(item);
    }
  }
}

File: contractstudy/src/test/java/contractstudy/collectContracts/common/StaticImportCollector/kotlin/testData/StaticImportClass.kt
package contractstudy.collectContracts.common.StaticImportCollector.kotlin.testData

import com.google.common.base.Preconditions
import org.jetbrains.annotations.NotNull
import java.util.List

class StaticImportClass {
    fun test(flag: Boolean): Boolean {
        return !flag
    }

    fun sum(num1: Int, num2: Int): Int {
        return num1 + num2
    }

    fun compute(area1: Int, area2: Int): Int {
        Preconditions.checkArgument(true, "This is an error")
        return sum(area1, area2)
    }

    @NotNull
    fun print(list: List<String>) {
        val class1 = StaticImportClass()
        for (item in list) {
            println(item)
        }
    }
}